internal/cli/cli.go

```diff
@@ -80,20 +80,26 @@ func readPrivateKey(fileName string) (privateKey [chacha20poly1305.KeySize]byte,
 		return
 	}
 	privateKey, err = keys.ReadPrivateKey(secretKeyFile, nil)
+	// nolint:nestif
 	if err != nil {
 		var password string
-		password, err = passwordPrompt("Enter the passphrase to unlock the key:")
-		if err != nil {
-			return
+		password, isPasswordSet := os.LookupEnv("C4GH_PASSPHRASE")
+
+		if !isPasswordSet {
+			password, err = passwordPrompt("Enter the passphrase to unlock the key:")
+			if err != nil {
+				return
+			}
 		}
+		fmt.Println(aurora.Yellow("Warning: Using a passphrase in an environment variable is considered insecure."))
```

Comment:
```suggestion
		} else {
		        fmt.Println(aurora.Yellow("Warning: Using a passphrase in an environment variable is considered insecure."))
		}
```

(formatting may be off).

Reply:
true, fixed in another place as well